client/scss/base/_typography.scss

```diff
@@ -24,6 +24,18 @@ h6 {
   margin-bottom: 0.6em;
 }
 
+h1 {
+  @include font('WB4');
+
+  &:first-child {
+    margin-top: 0;
+  }
+}
+
+h2 {
+  @include font('WB5');
+}
```

Comment:
Could we do something similar to what we did [here](https://github.com/wellcometrust/wellcomecollection.org/blob/master/client/scss/components/_body_content.scss#L4-L46) as this might wreak havoc elsewhere.